Standards/scs-XXXX-v1-volume-backup-service.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Volume Backup Functionality
+type: Standard
+status: Proposal
+track: IaaS
+---
+
+## Introduction
+
+OpenStack offers a variety of resources where users are able to transfer and store data in the infrastructure.
+A prime example of these resources are volumes which are attached to virtual machines as virtual block storage devices.
+As such they carry potentially large amounts of user data which is constantly changing at runtime.
+It is important for users to have the ability to create backups of this data in a reliable and effifcient manner.
+
+## Terminology
+
+| Term | Meaning |
+|---|---|
+| CSP | Cloud Service Provider, provider managing the OpenStack infrastructure |
+
+## Motivation
+
+The [volume backup functionality of OpenStack](https://docs.openstack.org/cinder/latest/admin/volume-backups.html) is a feature that is not available in all OpenStack clouds per default.
+The feature requires a backend to be prepared and configured correctly before it can be used.
+In Cinder, this is a separate configuration to the general storage backend of the volume service and is not mandatory.
```

Comment:
I rephrased this a bit more in the latest commit because if you remove the direct mention of OpenStack/Cinder here, then the statement cannot be as strict since other implementations may behave differently. Thus, I weakened the statement a bit accordingly.